src/Interactive/VbiCore/vbi.rsp

```diff
@@ -0,0 +1,35 @@
+/r:System.Collections.dll
```

Comment:
I suspect you just copying C# here - but let's get a bug filed on it, and not block your check-in. 

My statement: Response files do not make sense for CoreCLR, where references are different from what you run again.